modules/scripts/languages/en.json

```diff
@@ -0,0 +1,20 @@
+{
+    "SCRIPTS": {
+        "MODULE_DETAILS": {
+            "SINGULAR_NAME": "Script",
+            "PLURAL_NAME": "Scripts"
+        },
+        "FIELDS": {
```

Comment:
Add Japanese (ja.json), Korean (ko.json), and Simplified Chinese (zh_cn.json) JSONs for `Scripts` module